cmd/security-agent/subcommands/config/config.go

```diff
@@ -139,19 +140,13 @@ func getSettingsClient(_ *cobra.Command, _ []string) (settings.Client, error) {
 	return settingshttp.NewClient(c, apiConfigURL, "security-agent"), nil
 }
 
-func showRuntimeConfiguration(_ log.Component, _ config.Component, _ secrets.Component, params *cliParams) error {
-	c, err := params.getClient(params.command, params.args)
-	if err != nil {
-		return err
-	}
-
-	runtimeConfig, err := c.FullConfig()
+func showRuntimeConfiguration(_ log.Component, _ config.Component, _ secrets.Component, _ *cliParams) error {
+	runtimeConfig, err := fetcher.FetchSecurityAgentConfig(pkgconfig.Datadog)
```

Comment:
Could we use the `config.Component` rather the `pkgconfig`? 

Mostly to make sure we standardize the use of the config component